src/playdate.ts

```diff
@@ -25,7 +25,7 @@ export function runSimulator(
     case "win32":
       var simulator = `${sdkPath}\\bin\\PlaydateSimulator.exe`;
       return spawnSync("cmd", ["/c", simulator, output]);
-    case "darwin":
+    default:
       var simulator = `${sdkPath}/bin/Playdate Simulator.app`;
       return spawnSync("open", [simulator, output]);
```

Comment:
This would not work in linux - happy to have this all in, but you'll need to figure out what the paths for the simulator are on linux

Reply:
Oh geez, yup, easy mistake. Sorry I threw the commit togeter a little late at night. Incoming correction